integrations/openai/src/databricks_openai/mcp_server_tookit.py

```diff
@@ -0,0 +1,87 @@
+from typing import Callable, List
+
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.credentials_provider import OAuthCredentialsProvider
+from databricks_mcp import DatabricksMCPClient
+from openai.types.chat import ChatCompletionToolParam
+from pydantic import BaseModel
+
+
+class ToolInfo(BaseModel):
+    name: str
+    spec: ChatCompletionToolParam
+    exec_fn: Callable
+
+
+class McpServerToolkit:
+    def __init__(
+        self,
+        url: str = None,
+        app_name: str = None,
+        connection_name: str = None,
+        name: str = None,
```

Comment:
Same here, can we just take in url and name?

It is easier to just copy paste the URL from the MCP catalog page as well. So users don't have reason on which one to use and less code on our side as well for validations?